### ISSUE TITLE
Chromebook Pixel 2013 with i5 3427u , the CPU's max clock is 1.8 G hz,  the latest full rom  2019-12-01. No Turbo function ??

### ISSUE THREAD

**Issue by whmzsu** [MrChromebox/firmware#195]
Pixelbook 2013 with i5 3427u , the CPU's max clock is 1.8 G hz,  the latest full rom  2019-12-01

From the intela ark 
Processor Base Frequency 1.80 GHz
Max Turbo Frequency 2.80 GHz
Both under Ubuntu 19.10   or Windows 1909. 

Is it normal ???No Turbo Function ??  
BTW:Fully charged and with Power supply plugged in

**Comment by MrChromebox** [comment]
am able to reproduce here, am looking into it

**Comment by terrancewong** [comment]
Having same issue for years. sometimes CPU goes into turbo frequencies when playing video...

I tried hacking MSR 
```
# wrmsr --all  0x610 0x4c80c800FC80C8h
```

noluck

**Comment by MrChromebox** [comment]
yeah I'm not sure, I've had a few other ppl look at it too.

**Comment by Shercoolcool** [comment]
same issue with full rom 2020.06.06

**Comment by bpvarsity** [comment]
Can confirm the issue with my Chromebook Pixel 2013 as well

**Comment by bpvarsity** [comment]
I have not been able to find a kernel or distro that will let the cpu turbo. All of them seem to max at 1.8Ghz even though turbo shows to be enabled. Really wish I could find a way to get the turbo to work

**Comment by terrancewong** [comment]
but back some time in 2017~2018 I can run 2.4GHz on Ubuntu (16.04 or similar) XD

**Comment by bpvarsity** [comment]
> but back some time in 2017~2018 I can run 2.4GHz on Ubuntu (16.04 or similar) XD

Yeah I haven't gone back that far in versions but I can try it just to see if it can turbo
